packages/manager/modules/server-sidebar/src/sidebar.constants.js

```diff
@@ -16,7 +16,7 @@ export const DEDICATED_SERVER_CONFIG = {
     {
       path: '/dedicated/housing',
       category: 'HOUSING',
-      state: 'app.dedicated.housing',
+      state: 'dedicated-housing.dashboard',
```

Comment:
You should add `dedicated-housing` in parents `loadOnState` as the `dedicated housing` state move to root.